### PR TITLE
fixed port and updated version

### PR DESCRIPTION
**Pull request by lennard0711**
I updated the docker compose version from 2 => 3 and I fixed the wrong port variable

### CODE REVIEW

**Comment by dbarranco** [comment]
Hi @lennard7001 

First of all, thank you so much for your contribution.
I'm sorry, but I can not accept the contribution, as adding the `GAME_SERVER_LOGIN_TOKEN` will make the container don't work out-of-the-box.
By the moment I would prefer to leave the environment variables empty and let the user the option to configure them for public servers. Note that for private-LAN servers that env. var won't be needed.

If you remove that part, I will be happy to accept the update from the docker-compose v2 to v3.

Thanks again!
David

File: docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
-version: '2'
+version: '3'
 services:
   csgo:
-    image: 'dbaus/csgo:latest'
+    image: "dbaus/csgo:latest"
```
Comment (dbarranco) [inline docker-compose.yml]: Could you please leave this as single quotes?
Would like to have the convention along all the docker-compose file.

Thanks! :)

**Comment by lennard0711** [comment]
Ignore the useless comments in the middle pls. I tried to test something and I were stupid. And I removed the `MAINTAINER` thing since it's deprecated

File: Dockerfile [hunk 2]
```diff
@@ -1,33 +1,16 @@
-FROM bitnami/minideb:jessie
-
-MAINTAINER David Barranco <d.baus@outlook.com>
```
Comment (dbarranco) [inline Dockerfile]: As the MAINTAINER label it's deprecated, you could set:

```
LABEL maintainer="arp4@protonmail.com"
```

:D 
The rest LGTM!

**Review by dbarranco — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 3]
```diff
@@ -1,7 +1,11 @@
...
 services:
   csgo:
     image: 'dbaus/csgo:latest'
     ports:
-      - '27015/udp:27015/udp'
-
+      - '27015:27015/udp'
+      
+# If you want to create a public server remove the comments for the environment variable and add your Game Server Login Token
+# More infos over here: https://github.com/dbarranco/docker-csgo#environment-variables
```
Comment (dbarranco) [inline docker-compose.yml]: typo: `s/infos/info`

**Review by dbarranco — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 3 above)
Comment (lennard0711) [reply docker-compose.yml]: Ooops the german came through

**Comment by lennard0711** [comment]
Everything should be done now :)

**Comment by dbarranco** [comment]
Looks good to me!
Thanks for the contribution :)

**Pull request merged**